examples/bayesian_nn_analytic_kl.py

```diff
@@ -67,46 +67,46 @@ def mapping(self, x, z):
         Parameters
         -------
         x : tf.tensor
-            n_data x D
+            n_minibatch x D
 
         z : tf.tensor
-            num_vars
+            n_vars
 
         Returns
         -------
         tf.tensor
-            vector of length n_data
+            vector of length n_minibatch
         """
         h = x
         for W, b in self.unpack_weights(z):
             # broadcasting to do (h*W) + b (e.g. 40x10 + 1x10)
             h = self.nonlinearity(tf.matmul(h, W) + b)
 
-        h = tf.squeeze(h) # n_data x 1 to n_data
+        h = tf.squeeze(h) # n_minibatch x 1 to n_minibatch
         return h
 
     def log_lik(self, xs, zs):
         """Returns a vector [log p(xs | zs[1,:]), ..., log p(xs | zs[S,:])]."""
         x, y = xs['x'], xs['y']
         mus = tf.pack([self.mapping(x, z) for z in tf.unpack(zs)])
-        # broadcasting to do mus - y (n_minibatch x n_data - n_data)
+        # broadcasting to do mus - y (n_samples x n_minibatch - n_minibatch)
         log_lik = -tf.reduce_sum(tf.pow(mus - y, 2), 1) / self.lik_variance
         return log_lik
 
-def build_toy_dataset(n_data=40, noise_std=0.1):
+def build_toy_dataset(n_minibatch=40, noise_std=0.1):
```

Comment:
This one refers to the number of data points for building the toy data set. So maybe something like `N`?